main.cpp

```diff
@@ -1,7 +1,34 @@
 #include <iostream>
+#include <cstdint>
+#include "Points.hpp"
+/*
+enum class Result
+{
+   ADDED_SUCCESSFULLY,
+   ADDED_TO_MAX,
+   REMOVED_SUCESSFULLY,
+   REMOVED_TO_ZERO
+};
+*/
+
+using HP = Points;
+using MP = Points;
+
+struct Player
+{
+   HP _HP;
+   MP _MP;
+
+   Player(Points HP, Points MP)
+      : _HP(HP)
+      , _MP(MP)
+   {}
+};
 
 int main()
 {
-        std::cout << "Hello!" << std::endl;
+   Player p(100, 100);
+
+   std::cout << "Kaczka!" << std::endl;
 	return 0;
 }
```

Comment:
Tak to nic już nie znajduję, dobrze wykonany szkielet pod pointsy, jedynie dodałbym kilka testów które na przykładach sprawdziłyby funkcje operujące na Pointsach.